### PR TITLE
stabilize windows_process_extensions_raw_arg

### PR DESCRIPTION
**Pull request by Xaeroxe**
Stabilizes the feature tracked at https://github.com/rust-lang/rust/issues/92939

### CODE REVIEW

**Comment by m-ou-se** [comment]
FCP started here: https://github.com/rust-lang/rust/issues/92939#issuecomment-1062923488

File: library/std/src/os/windows/process.rs [hunk 1]
```diff
@@ -159,7 +159,7 @@ pub trait CommandExt: Sealed {
     ///
     /// This is useful for passing arguments to `cmd.exe /c`, which doesn't follow
     /// `CommandLineToArgvW` escaping rules.
-    #[unstable(feature = "windows_process_extensions_raw_arg", issue = "29494")]
+    #[stable(feature = "windows_process_extensions_raw_arg", since = "1.61.0")]
```
Comment (Xaeroxe) [inline library/std/src/os/windows/process.rs:162]: We should be sure to update this version number before we merge

**Comment by dtolnay** [comment]
- Bumped the `stable` attribute to 1.62 because #95636

**Review by dtolnay — APPROVED** [review]

**Pull request merged**